board/piksiv3/rootfs-overlay/etc/init.d/S84upgrade_tool

```diff
@@ -2,8 +2,9 @@
 
 start() {
   export FIRMWARE="/media/sda1/PiksiMulti-*.bin"
-  export LOGLEVEL="--info"
-
+  export LOGLEVEL="--warn"
+  # Sleep briefly to allow the automounting of flashdrive
+  sleep 0.5 
```

Comment:
I can poll with an ultimate timeout.  But we cannot block here because the user may be intending to upgrade.